### PR TITLE
Set enableConversions to true for PDF->PNG conversion

### PR DESCRIPTION
**Pull request by mserranom**
Setting was added in https://github.com/overleaf/filestore/pull/43.

### CODE REVIEW

**Review by JuneKelly — APPROVED** [review]

File: settings.coffee [hunk 1]
```diff
@@ -312,6 +312,11 @@ if process.env["SHARELATEX_PASSWORD_VALIDATION_PATTERN"] or process.env["SHARELA
 # ShareLaTeX Server Pro
 #######################
 
+# Enables template thumbnail generation via Filestore.
```
Comment (das7pad) [inline settings.coffee]: IIRC It's also used for the preview of PDFs in the file tree of a project.

-> which grants every user wild access on the conversion feature/vulnerability.

**Review by das7pad — CHANGES_REQUESTED** [review]

File: settings.coffee (hunk 1 above)
Comment (JuneKelly) [reply settings.coffee]: Good catch @das7pad, I think we're going to change this so it's set by a new environment-variable, disabled by default. If admins are using CE/SP in a private setting they can turn it on explicitely.

Comment (das7pad) [reply settings.coffee]: There is an environment variable for this already in place. See https://github.com/overleaf/filestore/commit/a441f46ed7fb2b958a6c642ddf64b53a38a0abf2

File: settings.coffee [hunk 2]
```diff
@@ -312,6 +317,7 @@ if process.env["SHARELATEX_PASSWORD_VALIDATION_PATTERN"] or process.env["SHARELA
 # ShareLaTeX Server Pro
 #######################
 
+
```
Comment (mserranom) [inline settings.coffee]:
```suggestion
```

**Comment by mserranom** [comment]
Closing as flag is set in https://github.com/overleaf/overleaf/pull/635/files using [filestore default config](https://github.com/overleaf/filestore/blob/master/config/settings.defaults.coffee#L53)

**Pull request closed without merging**